.github/workflows/node.js.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: set environment variables
+      uses: allenevans/set-env@v1.0.0
+      with:
+          MY_ENV_VAR: 'my value'
+          check: ${{ secrets.GITLAPUSER }}
+          super_secret: ${{ secrets.token }}
+    
+    - run: |
```

Comment:
This step was just in the CRR example file to simulate a test report. you don't want this.